redaxo/src/core/lib/console/application.php

```diff
@@ -11,4 +14,15 @@ public function __construct()
     {
         parent::__construct('REDAXO', rex::getVersion());
     }
+
+    public function doRun(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            return parent::doRun($input, $output);
+        } catch (\Exception $e) {
```

Comment:
Kannst du diesen catch block dann einfach weglassen?

Reply:
Dann werden Exceptions auch in FatalErrorExceptions gewandelt, Exception implementiert schließlich `\Throwable`. 
PS: Ja, ich füge einen Kommentar hinzu. ;)

Reply:
Danke ;)